From d7bf4dfa2a99cd82e27350620735c7ef3bf5fb69 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Tue, 28 Feb 2017 14:52:48 +0800
Subject: [PATCH 1647/5242] MLK-14286-1 ARM: dts: imx7ulp: add two new
 properties for mipi dsi node

commit  fc40560165484f641bdebe93586f6d5ff5b813d1 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add the 'data-lanes-num' and 'max-data-rate' properties which
are used to describe this mipi dsi capabilites.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
(cherry picked from commit bfe7d0e1c6931467f00a382e48aa592bf50c9339)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp.dtsi |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/imx7ulp.dtsi b/arch/arm/boot/dts/imx7ulp.dtsi
index afde8040..364b9f1 100644
--- a/arch/arm/boot/dts/imx7ulp.dtsi
+++ b/arch/arm/boot/dts/imx7ulp.dtsi
@@ -488,6 +488,8 @@
 			interrupts = <GIC_SPI 46 IRQ_TYPE_LEVEL_HIGH>;
 			clocks = <&clks IMX7ULP_CLK_DSI>;
 			clock-names = "mipi_dsi_clk";
+			data-lanes-num = <2>;
+			max-data-rate  = <800000000>;
 			sim = <&sim>;
 			status = "disabled";
 		};
-- 
1.7.9.5

